SQL_files/baseball_batting_average.sql

```diff
@@ -58,4 +58,32 @@ JOIN baseball.BATTERS_ROLLING AS b
 ON a.local_date WHERE DATEDIFF(a.local_date,b.local_date) BETWEEN 0 AND 100 AND a.batter=b.batter
 ORDER BY a.local_date ASC 
 LIMIT 0,20; # use join on itself instead of 2nd select
-*/
\ No newline at end of file
+*/
```

Comment:
Do you need both sql files here?

Reply:
I just had all my stuff from pervious hws so when I merge there isn't any conflicts